src/Services/Basket/Basket.API/Grpc/BasketService.cs

```diff
@@ -1,102 +1,93 @@
-﻿using Grpc.Core;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.eShopOnContainers.Services.Basket.API.Model;
-using Microsoft.Extensions.Logging;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace GrpcBasket
+﻿namespace GrpcBasket;
+public class BasketService : Basket.BasketBase
```

Comment:
```suggestion

public class BasketService : Basket.BasketBase
```

Add a space between `namespace` and `class` defs, for readability.